### DISCUSSION TITLE
Dark Mode default colors?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [dvtng/react-loading-skeleton#127] [Q&A] [answered]
I'd like to know if you've seen examples of any dark mode default colors that look good enough?

I have been Googling but the results don't understand what I want to search. Currently, doing trial & error but one perfect combo would be good enough for me!

**Comment by srmagura** [comment, marked as answer]
Haha, I do not know if there is a "perfect combo". Here are some colors that look OK in my opinion:

```ts
const baseColor = '#c0c0c0'
const highlightColor = '#a0a0a0'
```

If it's relevant, we have a light/dark theme example in our Storybook [here](https://github.com/dvtng/react-loading-skeleton/blob/master/src/__stories__/SkeletonTheme.stories.tsx#L42-L77).

**Reply by deadcoder0904** [reply]
@srmagura naah, that looks dull on dark mode. it has to be dark but must show loading animation as well. i just can't seem to find a nice combo for so long 😂

**Reply by deadcoder0904** [reply]
@srmagura the storybook has nice combo. you've probably wanted to mention that but used light colors 😂

anyways this looks decent enough:

```tsx
<SkeletonTheme baseColor="#333" highlightColor="#999">
	{children}
</SkeletonTheme>
```

but i went with:

```tsx
<SkeletonTheme baseColor="#111111" highlightColor="#444444">
	{children}
</SkeletonTheme>
```

naah, i think this one looks much better even on text:

```tsx
<SkeletonTheme baseColor="#333" highlightColor="#666">
	{children}
</SkeletonTheme>
```
